test/preset-mini.test.ts

```diff
@@ -353,4 +353,30 @@ describe('preset-mini', () => {
         .text-sm{font-size:0.875rem;line-height:1.25rem;}"
       `)
   })
+
+  it('border(-[trlb])?-[a_b_c]', async () => {
+    const uno = createGenerator({
```

Comment:
Also you can add the test token to the [preset mini target file](https://github.com/unocss/unocss/blob/main/test/assets/preset-mini-targets.ts).

Reply:
good idea